src/Policy/CallbackPolicyIncludingPath.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Chubbyphp\Serialization\Policy;
+
+use Chubbyphp\Serialization\Normalizer\NormalizerContextInterface;
+
+final class CallbackPolicyIncludingPath implements PolicyInterface
+{
+    /**
+     * @var callable
+     */
+    private $callback;
+
+    public function __construct(callable $callback)
+    {
+        $this->callback = $callback;
+    }
+
+    /**
+     * @deprecated
+     */
+    public function isCompliant(NormalizerContextInterface $context, object $object): bool
+    {
+        @trigger_error('Use "isCompliantIncludingPath()" instead of "isCompliant()"', E_USER_DEPRECATED);
```

Comment:
exception (https://github.com/chubbyphp/chubbyphp-serialization/blob/master/src/SerializerLogicException.php)
```
@trigger_error('Use "isCompliantIncludingPath()" instead of "isCompliant()"', E_USER_DEPRECATED);

        return $this->isCompliantIncludingPath('', $object, $context);
```